okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -491,6 +498,11 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
         + "   \"description\" : \"get modules that provide some interface\" "
         + " }, "
         + " { "
+        + "   \"permissionName\" : \"okapi.import.modules.post\", "
+        + "   \"displayName\" : \"Okapi - import modules\", "
+        + "   \"description\" : \"Import modules\" "
+        + " }, "
```

Comment:
Can you copy the wording from the .raml file into the descriptor?

The existing permission

    "permissionName" : "okapi.proxy.modules.post"
    "displayName" : "Okapi - declare a module"
    "description" : "Declare a module"

should become

    "permissionName" : "okapi.proxy.modules.post"
    "displayName" : "Okapi - announce new module to proxy"
    "description" : "Announce a new module to the proxy"

Reply:
ok.